charts/pega/values.yaml

```diff
@@ -50,17 +50,17 @@ global:
     customerDataSchema: ""
 
   customArtifactory:
-    # If using a custom artifactory to download JDBC uri, supply the authentication details here.
-    auth:
+    # If using a custom artifactory secured with Basic or APIKey Authentication to download JDBC driver, supply the authentication details here.
+    authentication:
       # Provide either basic authentication or api key authentication details based on preferred way of authentication mechanism to custom artifactory.
       basic:
         username: "YOUR_ARTIFACTORY_USERNAME"
         password: "YOUR_ARTIFACTORY_PASSWORD"
       apiKey:
         headerName: "YOUR_ARTIFACTORY_DEDICATED_APIKEY_HEADER"
         value: "YOUR_ARTIFACTORY_API_KEY"
-    enable_ssl_verification: true
-    #Custom artifactory domain server certificate if needed.
+    enableSSLVerification: true
```

Comment:
Leave customArtifactory.enableSSLVerification enabled to ensure secure access to your custom artifactory; when customArtifactory.enableSSLVerification is false, SSL verification is skipped and establishes an insecure connection.

enableSSLVerification: true

Provide a required domain certificate for your custom artifactory server; if none is required, leave this field blank.
certificate: